Traceback (most recent call last):
  File "/home01/x2889a02/GoodLiar/main_on_the_fly.py", line 141, in <module>
    main()
  File "/home01/x2889a02/GoodLiar/main_on_the_fly.py", line 96, in main
    default_config['train']['tracker'] = None
NameError: name 'default_config' is not defined
Traceback (most recent call last):
  File "/home01/x2889a02/GoodLiar/main_on_the_fly.py", line 141, in <module>
    main()
  File "/home01/x2889a02/GoodLiar/main_on_the_fly.py", line 96, in main
    default_config['train']['tracker'] = None
NameError: name 'default_config' is not defined